packages/@webex/plugin-meetings/src/meeting-info/index.ts

```diff
@@ -71,11 +71,19 @@ export default class MeetingInfo {
    * @memberof MeetingInfo
    */
   private requestFetchInfo(options: any) {
-    const {meetingId} = options;
+    const {meetingId, sendCAevents} = options;
     if (meetingId) {
```

Comment:
in this case, check if sendCAevents is true and send both, removing the meetingId